haunted-houses/haunted-houses-persistence/src/main/java/cz/muni/fi/pa165/hauntedhouses/model/GameInstance.java

```diff
@@ -0,0 +1,71 @@
+package cz.muni.fi.pa165.hauntedhouses.model;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+/**
+ * @author David Hofman
+ */
+@Entity
+public class GameInstance {
+    @Id
+    @GeneratedValue(strategy= GenerationType.SEQUENCE)
+    private Long id;
+
+    @Column(nullable=false)
+    private int banishesAttempted;
+
+    @Column(nullable=false)
+    private int banishesRequired;
+
+    @OneToOne
+    @JoinColumn
+    private Player player;
```

Comment:
Toto je OK, ale to isté treba spraviť aj so Specter, keďže jedna GameInstance má vždy jedného ducha. Navyše treba GameInstance pridať aj do Specter a do Player, aby sme sa vedeli aj zo strany hráča a ducha dostať k hre. Nezabudni na get/set